src/modules/temperature.cpp

```diff
@@ -53,20 +53,24 @@ namespace modules {
   bool temperature_module::update() {
     m_temp = std::strtol(file_util::contents(m_path).c_str(), nullptr, 10) / 1000.0f + 0.5f;
     int temp_f = floor(((1.8 * m_temp) + 32) + 0.5);
+    int temp_k = m_temp + 273;
 
     string temp_c_string = to_string(m_temp);
     string temp_f_string = to_string(temp_f);
+    string temp_k_string = to_string(temp_k);
 
     // Add units if `units = true` in config
     if(m_units) {
       temp_c_string += "°C";
       temp_f_string += "°F";
+      temp_k_string += "K";
```

Comment:
Should probably include the degree sign

Reply:
[kelvin has no degree sign](https://en.wikipedia.org/wiki/Kelvin)

Reply:
Well TIL. Maybe add a comment in the code (doesn't require a link)

Reply:
i dont think it is reqired since it is cleary defined like that

Reply:
Well, it doesn't have to be explained for °C to have a degree sign, so why should it be needed to explain for Kelvin not having a degree sign? What do Temperatures have to do with Angles in the first place? Even angles don't need a degree sign if you use Radians.